stable/application-mesh/templates/envoy-filter.yaml

```diff
@@ -1,36 +1,26 @@
 {{- if .Values.global.mesh.enabled }}
-{{- range $_, $ingress := .Values.ingress }}
-{{- if ne $ingress.host "example.matic.com" }}
-{{- if $ingress.settings }}
-{{- if $ingress.settings.max_request_bytes }}
-{{- $host := required "Ingress host must be specified" $ingress.host }}
+{{- range $_, $destination := .Values.destinations }}
+{{- if $destination.max_request_bytes }}
 ---
 apiVersion: networking.istio.io/v1alpha3
 kind: EnvoyFilter
 metadata:
-  name: {{ include "application-helpers.name" $ }}-{{ include "application-helpers.uniq-id" $host }}
+  name: {{ include "application-helpers.name" $ }}-{{ $destination.component }}
```

Comment:
If we create a filter per destination - then the name should be something like `amp-webserver-request-size`. As we might add more filters in the future